Assets/Scripts/UI/TitleScreenScript.cs

```diff
@@ -0,0 +1,113 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+using System.Collections;
+
+public class TitleScreenScript : MonoBehaviour {
+
+    public Button newGameButton;
+    public Button continueButton;
+    public Button noButton;
+    public Image background;
+    public AnimationClip warningAnimation;
+    private Animator titleAnimator;
+    private bool isGameSaved;
+
+    static int logoFadeStateHash;
+    static int doneStateHash;
+
+    // Use this for initialization
+    void Start () {
+        //Get animator and set the animation states hashes
+        titleAnimator = GetComponent<Animator>();
+        logoFadeStateHash = Animator.StringToHash("Base Layer.Fade In Logo");
+        doneStateHash = Animator.StringToHash("Base Layer.AllDone");
+
+        isGameSaved = checkIfGameIsSaved();
+        continueButton.interactable = isGameSaved;
+
+        //Start the coroutine that will wait for the animation states
+        StartCoroutine(waitForLogoAndStartMusic());
+        StartCoroutine(waitForNewGameButtonAndSelect());
+        titleAnimator.SetBool("isGameReady", true);
+    }
+
+   
+    //Start playing the music when the logo starts fading in
+    IEnumerator waitForLogoAndStartMusic() {
+        while(titleAnimator.GetCurrentAnimatorStateInfo(0).fullPathHash != logoFadeStateHash) {
+            Debug.Log("Still waiting");
+            yield return null;
+        }
+       ThemeMusicManager.getInstance().playThemeSong("TitleScreen");
```

Comment:
so if ur playing the song here, don't do anything else, just go to ThemeMusicManager and fade it out whenever it's playing in the beginning of playThemeSong method, need to be the beginning cus you can go to any scenes from the title scene